imageio/core/fetching.py

```diff
@@ -140,26 +141,36 @@ def _fetch_file(url, file_name, print_destination=True):
     print_destination: bool, optional
         If true, destination of where file was saved will be printed after
         download finishes.
+    proxy_address: bool, optional
+        If false, no proxy employed; if ,
     resume: bool, optional
         If true, try to resume partially downloaded files.
     """
     # Adapted from NISL:
     # https://github.com/nisl/tutorial/blob/master/nisl/datasets.py
-    
+
     print('Imageio: %r was not found on your computer; '
           'downloading it now.' % os.path.basename(file_name))
-    
+
     temp_file_name = file_name + ".part"
     local_file = None
     initial_size = 0
     errors = []
     for tries in range(4):
         try:
-            # Checking file size and displaying it alongside the download url
-            remote_file = urlopen(url, timeout=5.)
+            if proxy_address:
```

Comment:
This bit only works with python 3 (because of the urllib module). We can either try to also make it work with Python 2, or put just a check here and raise an error on Python 2. I'd prefer the latter, because it keeps the code simpler, and end of life for Python 2 is nearing.

Reply:
What's your preferred method for raising Python2 flags in this repo?

```raise RuntimeError('Python2+ proxy unsupported') from error```

or is there a class of errors I can draw from?

Reply:
RuntimeError with a clear message should do :)